packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -79,6 +79,7 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{ sync: DashboardCursor
       scaleKey: xScaleKey,
       isTime: true,
       placement: AxisPlacement.Bottom,
+      label: xField.config.custom?.xAxisLabel,
```

Comment:
Is there a pure requirement for the label to be a separate property? I think we should strive to use overrides rather than introducing this additional option. This PR would then turn into a two-liner:)

Reply:
Hey, that's a great idea, thanks @dprokop